sql/compact/src/main/java/WithExistingClassExample.java

```diff
@@ -0,0 +1,57 @@
+import com.hazelcast.config.CompactSerializationConfig;
+import com.hazelcast.config.Config;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.map.IMap;
+import com.hazelcast.sql.SqlResult;
+import com.hazelcast.sql.SqlRow;
+import com.hazelcast.sql.SqlService;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Demonstrates usage of Compact(BETA) format with SQL Service with an existing DTO class
+ */
+public class WithExistingClassExample {
+
+    public static void main(String[] args) throws InterruptedException {
+        Config config = new Config();
+        CompactSerializationConfig compactSerializationConfig = config.getSerializationConfig().getCompactSerializationConfig();
+        compactSerializationConfig.setEnabled(true);
+        config.getJetConfig().setEnabled(true);
+        HazelcastInstance hazelcast = Hazelcast.newHazelcastInstance(config);
+
+        IMap<Integer, Person> myMap = hazelcast.getMap("myMap");
+        myMap.put(1, new Person(1, "John", "Stone"));
+
+        SqlService sqlService = hazelcast.getSql();
+
+        sqlService.execute("CREATE MAPPING myMap ( " +
```

Comment:
A comment saying that this reuses existing schema since the fields are the same.